extensions/mssql/src/constants.ts

```diff
@@ -10,3 +10,53 @@ export const serviceCrashMessage = 'SQL Tools Service component exited unexpecte
 export const serviceCrashButton = 'View Known Issues';
 export const serviceCrashLink = 'https://github.com/Microsoft/vscode-mssql/wiki/SqlToolsService-Known-Issues';
 export const extensionConfigSectionName = 'mssql';
+
+// DATA PROTOCOL VALUES ///////////////////////////////////////////////////////////
+export const mssqlClusterProviderName = 'mssqlCluster';
+export const hadoopKnoxEndpointName = 'Knox';
+export const protocolVersion = '1.0';
+export const hostPropName = 'host';
+export const userPropName = 'user';
+export const knoxPortPropName = 'knoxport';
+export const passwordPropName = 'password';
+export const groupIdPropName = 'groupId';
+export const defaultKnoxPort = '30443';
+export const groupIdName = 'groupId';
+export const sqlProviderName = 'MSSQL';
+export const dataService = 'Data Services';
+
+export const hdfsHost = 'host';
+export const hdfsUser = 'user';
+export const UNTITLED_SCHEMA = 'untitled';
+
+export const mssqlProviderId = 'MSSQL';
```

Comment:
Please remove this and change reference to be `export const providerId = 'MSSQL';` since it's already in the extension